server/tilesource/base.py

```diff
@@ -287,58 +548,21 @@ def getRegion(self, width=None, height=None, **kwargs):
         :param height: maximum height in pixels.
```

Comment:
`width` and `height` are the output resolution.  I'll improve the documentation.